content/docs/guides/parallel-execution.md

```diff
@@ -0,0 +1,389 @@
+---
+title: Parallel Execution
+subtitle: Using multiple threads to reduce test times
+polyglot:
+ - java
+ - kotlin
+ 
+weight: 1800
+---
+{{% block "java,kotlin" %}}
+Cucumber-JVM allows [parallel execution](https://cucumber.io/blog/announcing-cucumber-jvm-4-0-0/) across multiple threads since version 4.0.0. There are several options to incorporate this built-in feature in a Cucumber project.
+
+- [JUnit](/docs/guides/parallel-execution/#junit)
+- [TestNG](/docs/guides/parallel-execution/#testng)
+- [CLI](/docs/guides/parallel-execution/#cli)
+
+For each of these options, this tutorial will look at the project setup, configuration settings and execution commands.
+{{% /block %}}
+
+# JUnit
+
+{{% block "java,kotlin" %}}
+Cucumber can be executed in parallel using **JUnit and Maven test execution plugins**. In JUnit the **feature files are run in parallel rather than scenarios**, which means **all the scenarios in a feature file will be executed by the same thread**. You can use either Maven Surefire or Failsafe plugin to execute the runners.
+
+- Create a Maven project in your favorite IDE. Add `cucumber-java8` and `cucumber-junit` **dependencies** to the `POM`.
+
+```shell
+<dependency>
+	<groupId>io.cucumber</groupId>
+	<artifactId>cucumber-java8</artifactId>
+	<version>{{% version "cucumberjvm" %}}</version>
+	<scope>test</scope>
+</dependency>
+<dependency>

```

Comment:
We also mention this [here](https://cucumber.io/docs/cucumber/checking-assertions/#junit). Again, I think it's ok to duplicate, otherwise the reader would be clicking back and forth.

Note that the JUnit dependency itself is missing on this page - could you add it here too?

Reply:
No need to add the junit or testng dependency explicitly in pom. The cucumber-junit or cucumber-testng pulls this as a transitive one.

Reply:
Though the cucumber-junit dependency is missing at this [link](https://cucumber.io/docs/installation/java/#maven).